tests/deleteobserver.js

```diff
@@ -28,6 +29,12 @@ describe( 'DeleteObserver', () => {
 	} );
 
 	describe( 'delete event', () => {
+		const initialEnvMac = env.isMac;
+
+		afterEach( () => {
+			env.isMac = initialEnvMac;
```

Comment:
Can we use `testUtils.createSinonSandbox()` instead? Because this property is overriden completely manually, when reading the below tests I couldn't understand whether (and how) it's going to be restored after tests.